app/controllers/v1/calls_controller.rb

```diff
@@ -0,0 +1,10 @@
+class V1::CallsController < V1::BaseController
+
+  def create
+    person = Person.create_with(phone: params[:phone])).find_or_create_by(email: params[:email])
+    twilio_call = Integration::Twilio.initiate_congress_calling(phone: params[:phone])
+    call = person.calls.create(remote_id: twilio_call.sid)
```

Comment:
this should check to see if the person is already on a call. I think we said the frontend would call this method each time the user clicked 'Next'